src/taskgraph/run-task/run-task

```diff
@@ -1081,6 +1081,12 @@ def main(args):
         b"setup",
         b"run-task started in %s\n" % os.environ["TASK_WORKDIR"].encode("utf-8"),
     )
+    #logging command which envoked the script
+    calling_script = sys.argv[0]
```

Comment:
We don't want to just log the script, but rather the full command line. We should also use `args` here rather than `sys.argv` in case something called this `main` function programmatically.